### PR TITLE
Update the tekton release tasks for the imageDigestExporter image.

### PR DESCRIPTION
**Pull request by dlorenc**
# Changes

The first time I release v0.4.0 it failed because ko produces a new image
not in the configuration.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: tekton/publish-run.yaml [hunk 1]
```diff
@@ -142,7 +152,7 @@ spec:
         name: tekton-bucket
     params:
     - name: versionTag
-      value: v0.0.0 # REPLACE with the version you want to release, for nightly releases, we might want to use `vYYYYMMDD-commit`
+      value: v0.4.0 # REPLACE with the version you want to release, for nightly releases, we might want to use `vYYYYMMDD-commit`
```
Comment (vdemeester) [inline tekton/publish-run.yaml]: Should we keep this one as `v0.0.0` (to not overwrite the v0.4.0 images by mistake later on)

File: tekton/publish-run.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ spec:
   - name: url
     value: https://github.com/tektoncd/pipeline # REPLACE with your own fork
   - name: revision
-    value: master # REPLACE with your own commit
+    value: 9c52c698022faef09311093b1621b17cf903a8c0 # REPLACE with your own commit
```
Comment (vdemeester) [inline tekton/publish-run.yaml]: Same here :sweat_smile:

Comment (dlorenc) [reply tekton/publish-run.yaml]: Yup, sorry!

**Comment by dlorenc** [comment]
Fixed the issues from before.

**Comment by vdemeester** [comment]
@nader-ziada yaml failure seems legit

```
I0604 13:29:36.672] - apiVersion: tekton.dev/v1alpha1
I0604 13:29:36.672]   kind: TaskRun
I0604 13:29:36.672]   metadata:
I0604 13:29:36.672]     annotations:
I0604 13:29:36.672]       kubectl.kubernetes.io/last-applied-configuration: |
I0604 13:29:36.672]         {"apiVersion":"tekton.dev/v1alpha1","kind":"TaskRun","metadata":{"annotations":{},"name":"build-push-run","namespace":"default"},"spec":{"inputs":{"params":[{"name":"pathToDockerFile","value":"Dockerfile"},{"name":"pathToContext","value":"/workspace/workspace/examples/microservices/leeroy-web"}],"resources":[{"name":"workspace","resourceRef":{"name":"skaffold-git"}}]},"outputs":{"resources":[{"name":"builtImage","resourceRef":{"name":"skaffold-image-leeroy-web"}}]},"taskRef":{"name":"build-push-kaniko"}}}
I0604 13:29:36.672]     creationTimestamp: "2019-06-04T13:25:52Z"
I0604 13:29:36.672]     generation: 2
I0604 13:29:36.673]     labels:
I0604 13:29:36.673]       tekton.dev/task: build-push-kaniko
I0604 13:29:36.673]     name: build-push-run
I0604 13:29:36.673]     namespace: default
I0604 13:29:36.673]     resourceVersion: "5491"
I0604 13:29:36.673]     selfLink: /apis/tekton.dev/v1alpha1/namespaces/default/taskruns/build-push-run
I0604 13:29:36.673]     uid: 4681c8f0-86cc-11e9-933e-42010a800fc9
I0604 13:29:36.673]   spec:
I0604 13:29:36.673]     inputs:
I0604 13:29:36.673]       params:
I0604 13:29:36.673]       - name: pathToDockerFile
I0604 13:29:36.673]         value: Dockerfile
I0604 13:29:36.673]       - name: pathToContext
I0604 13:29:36.673]         value: /workspace/workspace/examples/microservices/leeroy-web
I0604 13:29:36.674]       resources:
I0604 13:29:36.674]       - name: workspace
I0604 13:29:36.674]         resourceRef:
I0604 13:29:36.674]           name: skaffold-git
I0604 13:29:36.674]     outputs:
I0604 13:29:36.674]       resources:
I0604 13:29:36.674]       - name: builtImage
I0604 13:29:36.674]         resourceRef:
I0604 13:29:36.674]           name: skaffold-image-leeroy-web
I0604 13:29:36.674]     taskRef:
I0604 13:29:36.674]       kind: Task
I0604 13:29:36.674]       name: build-push-kaniko
I0604 13:29:36.674]   status:
I0604 13:29:36.674]     conditions:
I0604 13:29:36.675]     - lastTransitionTime: "2019-06-04T13:25:59Z"
I0604 13:29:36.675]       message: 'invalid input resources: TaskRun''s declared resources didn''t match
I0604 13:29:36.675]         usage in Task: Didn''t provide required values: [workspace]'
I0604 13:29:36.675]       reason: TaskRunValidationFailed
I0604 13:29:36.675]       status: "False"
I0604 13:29:36.675]       type: Succeeded
I0604 13:29:36.675]     podName: build-push-run-pod-e0c5ca
I0604 13:29:36.675]     startTime: "2019-06-04T13:25:53Z"
```

**Comment by bobcatfish** [comment]
Something weird is going on:

```
I0604 13:29:36.675]       message: 'invalid input resources: TaskRun''s declared resources didn''t match
I0604 13:29:36.675]         usage in Task: Didn''t provide required values: [workspace]'
```

But in the TaskRun:

```
I0604 13:29:36.674]       resources:
I0604 13:29:36.674]       - name: workspace
I0604 13:29:36.674]         resourceRef:
I0604 13:29:36.674]           name: skaffold-git
```

Kinda reminds me of #929, still not 100% sure what was going wrong there :thinking:

**Comment by bobcatfish** [comment]
```
I0612 15:31:19.444] 2019/06/12 15:31:19 gcr.io/tekton-prow-0/tpipeline-e2e-img/controller-10a3e32792f33651396d02b6855a6e36:latest: digest: sha256:322c1a9a44075e780f3a9bca2728a710ac17fc081b0fb60cc66c8e456529b7da size: 754
I0612 15:31:19.445] 2019/06/12 15:31:19 Published gcr.io/tekton-prow-0/tpipeline-e2e-img/controller-10a3e32792f33651396d02b6855a6e36@sha256:322c1a9a44075e780f3a9bca2728a710ac17fc081b0fb60cc66c8e456529b7da
I0612 15:31:19.445] 2019/06/12 15:31:19 error processing import paths in "config/controller.yaml": Get https://gcr.io/v2/: net/http: TLS handshake timeout
```

:thinking: 

/test pull-tekton-pipeline-integration-tests

**Pull request merged**